.github/workflows/helm-template-generate-k8s-manifests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Generate K8S Manifests from Helm Charts
+
+on:
+  # Run this when PR operations are done
+  pull_request:
+    types: [ opened, synchronize, reopened, edited ]
+
+jobs:
+  generate-k8s-manifests:
+    name: Generate K8S manifests via `helm template` command
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kube-tag:
+          - v1.18.2
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.6.2
+
+      - name: Generate manifest for cluster-prep
+        run: |
+          pushd helm/conjur-config-cluster-prep > /dev/null
+            mkdir -p generated
+
+            helm template cluster-prep . \
+              -f user-values.yaml \
```

Comment:
Would it be better to use a name other than `user-values.yaml` that indicates how it's used, maybe `sample-values.yaml` or something like that?

Reply:
Done